soyjs/exec.go

```diff
@@ -406,6 +396,9 @@ func (s *state) visitPrint(node *ast.PrintNode) {
 			// no implementation, they just serve as a marker to cancel autoescape.
 		default:
```

Comment:
Will our custom directives get added into soyjs.PrintDirectives? If not, it will never get to this default case.